style.css

```diff
@@ -33,11 +33,12 @@
 }
 
 .hamburger {
-  background-image: url("./img/Hamburger.png");
+  /* background-image: url("./img/Hamburger.png"); */
```

Comment:
- [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile:

Reply:
all commented files removed.Clean code now available